packages/core/e2e/examples/cloudflare-worker-agent/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@llamaindex/cloudflare-worker-agent-test",
+  "version": "0.0.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "deploy": "wrangler deploy",
+    "dev": "wrangler dev",
+    "build": "wrangler deploy --dry-run --outdir dist",
+    "start": "wrangler dev",
+    "test": "vitest",
+    "cf-typegen": "wrangler types"
+  },
+  "devDependencies": {
+    "@cloudflare/vitest-pool-workers": "^0.2.3",
+    "@cloudflare/workers-types": "^4.20240423.0",
+    "typescript": "^5.4.5",
+    "vitest": "1.3.0",
+    "wrangler": "^3.52.0"
+  },
+  "dependencies": {
+    "llamaindex": "workspace:*"
```

Comment:
might be a bit confusing that cloudflare is using `llamaindex` and for edge we have a dedicated package. 
Technically, we don't have a reason for a dedicated edge package; the idea was that we could track usage separately.

Reply:
I think this use same package name will be better, It's not a suggeseted way to seperate pacakges